app/styles/ui/onboarding-tutorial/_nudge-arrow.scss

```diff
@@ -0,0 +1,70 @@
+.nudge-arrow {
+  // all arrows need this base styling
+  &::after {
```

Comment:
i used the `::after` pseudo-class here because these arrows are not really an intractable part of the interface, but an overlay.